benchmark/EU/assets-data.csv

```diff
@@ -1,406 +1,406 @@
-RO_Wind_Onshore,producer,true,false,false,1040,,5255,5255,0,,false,0,0,0,0,false,0,,0,false,,,
-RO_Battery,storage,true,false,false,4733,,100,100,0,,false,100,300,150,3,false,0,,0,false,,,
-RO_OCGT,producer,true,false,false,435,,1015,1015,0,,false,0,0,0,0,false,0,,0,false,,,
-RO_Pump_Hydro_Open,storage,true,false,false,0,,839,839,0,,false,839,1506400,753200,0,false,0,,0,false,,,
-RO_E_Balance,hub,true,false,false,0,,0,0,0,,false,0,0,0,0,false,0,,0,false,,,
-RO_E_ENS,producer,true,false,false,0,,10352,10352,0,,false,0,0,0,0,false,0,,0,false,,,
-RO_Hydro_Reservoir,storage,true,false,false,0,,2579,2579,0,,false,2579,2425300,1212650,0,false,0,,0,false,,,
-RO_Hydro_Reservoir_Inflow,producer,true,false,false,0,,2579,2579,0,,false,0,0,0,0,false,0,,0,false,,,
-RO_Pump_Hydro_Open_Inflow,producer,true,false,false,0,,839,839,0,,false,0,0,0,0,false,0,,0,false,,,
-UA_E_Demand,consumer,true,false,false,0,,0,0,1454,,false,0,0,0,0,false,0,,0,false,,,
-UA_Coal,producer,true,false,false,0,,836,836,0,,false,0,0,0,0,false,0,,0,false,,,
-UA_Hydro,producer,true,false,false,0,,14,14,0,,false,0,0,0,0,false,0,,0,false,,,
-UA_Solar,producer,true,false,false,870,,528,528,0,,false,0,0,0,0,false,0,,0,false,,,
-UA_Wind_Onshore,producer,true,false,false,1040,,235,235,0,,false,0,0,0,0,false,0,,0,false,,,
-UA_E_Balance,hub,true,false,false,0,,0,0,0,,false,0,0,0,0,false,0,,0,false,,,
-UA_E_ENS,producer,true,false,false,0,,1454,1454,0,,false,0,0,0,0,false,0,,0,false,,,
-UA_Hydro_Reservoir,storage,true,false,false,0,,55,55,0,,false,55,100,50,0,false,0,,0,false,,,
-UA_Hydro_Reservoir_Inflow,producer,true,false,false,0,,55,55,0,,false,0,0,0,0,false,0,,0,false,,,
-CY_E_Demand,consumer,true,false,false,0,,0,0,1331,,false,0,0,0,0,false,0,,0,false,,,
-CY_Gas,producer,true,false,false,830,,1210,1210,0,,false,0,0,0,0,false,0,,0,false,,,
-CY_Solar,producer,true,false,false,870,,800,800,0,,false,0,0,0,0,false,0,,0,false,,,
-CY_Wind_Onshore,producer,true,false,false,1040,,198,198,0,,false,0,0,0,0,false,0,,0,false,,,
-CY_Battery,storage,true,false,false,4733,,41,41,0,,false,41,123,62,3,false,0,,0,false,,,
-CY_E_Balance,hub,true,false,false,0,,0,0,0,,false,0,0,0,0,false,0,,0,false,,,
-CY_E_ENS,producer,true,false,false,0,,1331,1331,0,,false,0,0,0,0,false,0,,0,false,,,
+,{producer;consumer;storage;hub;conversion},{true;false},{group name},{true;false},{true;false},kEUR/MW/year,MW,MW,MW,MW,{empty;==;>=},{true;false},MWh/year,MWh,MWh,h,{true;false},kEUR/MWh/year,MWh,MWh,{true;false},{missing;binary;relaxed_binary},MWh,MWh
+name,type,active,,investable,investment_integer,investment_cost,investment_limit,capacity,initial_capacity,peak_demand,consumer_balance_sense,is_seasonal,storage_inflows,initial_storage_capacity,initial_storage_level,energy_to_power_ratio,storage_method_energy,investment_cost_storage_energy,investment_limit_storage_energy,capacity_storage_energy,investment_integer_storage_energy,use_binary_storage_method,max_energy_timeframe_partition,min_energy_timeframe_partition
```

Comment:
Shouldn't there be a column name here?

Reply:
It is missing the column name. Well spotted! Thanks 😄